requirements/dev-sdw-requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
```

Comment:
Python 3.9 is our default now that we target Bullseye in production. As above, I'd prefer if this file was re-generated using Puthon 3.9.